### PR TITLE
Jupyter Notebook for PyDP & Scikit-Learn comparison 

### PR DESCRIPTION
**Pull request by grilhami**
## Description
Added a Jupyter Notebook that compares the accuracy result of training a dataset using Scikit-Learn's `GaussianNB` and PyDP's `GaussianNB`; the latter which adding differential privacy mechanisms. 

## Affected Dependencies
`matplotlib` was added in the `pyproject.toml` file to allow the notebook to function properly.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -7,6 +7,7 @@ license = "Apache-2.0"
 
 [tool.poetry.dependencies]
 python = "^3.6"
+matplotlib = "^3.3.4"
```
Comment (chinmayshah99) [inline pyproject.toml]: is this necessary over here? If not, we can move it to dev dependencies

**Review by chinmayshah99 — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (grilhami) [reply pyproject.toml]: It's to make sure that the notebook runs well. But moving it to dev dependencies also sounds good, I'll make the changes

Comment (chinmayshah99) [reply pyproject.toml]: Moving it to the main dependency means the user requires this to install matplotlib when using pydp. This is not desirable behavior.

Comment (grilhami) [reply pyproject.toml]: I've already moved matplotlib as a dev dependency

**Review by chinmayshah99 — APPROVED** [review]

**Pull request merged**